contract_manager/scripts/common.ts

```diff
@@ -181,3 +181,37 @@ export function getSelectedChains(argv: {
   }
   return selectedChains;
 }
+
+/**
+ * Finds the entropy contract for a given EVM chain.
+ * @param chain The EVM chain to find the entropy contract for.
+ * @returns The entropy contract for the given EVM chain.
+ * @throws an error if the entropy contract is not found for the given EVM chain.
+ */
+export function findEntropyContract(chain: EvmChain): EvmEntropyContract {
+  for (const contract of Object.values(DefaultStore.entropy_contracts)) {
+    if (
+      contract instanceof EvmEntropyContract &&
```

Comment:
I don't think this condition is necessary

Reply:
Yeah, right